RPIPs/RPIP-28.md

```diff
@@ -0,0 +1,47 @@
+---
+rpip: 28
+title: Deposits Under the Minimum
+description: Allow minipool deposits that include an appropriate RPL stake, even when under the "minimum" RPL threshold 
+author: Valdorff (@Valdorff)
+discussions-to: 
+status: Draft
+type: Protocol
+category: Core
+created: 2023-08-11
+---
+
+## Abstract
+
+Right now, if an NO falls below the minimum (value of RPL staked <10% of borrowed ETH), they can't 
```

Comment:
typo ("an NO" -> "a NO")

Reply:
unless this is because you pronounce it en oh?

Reply:
I do pronounce it "en oh" 😛 

Language is silly in that this matters, but here we are

Reply:
either article works for me :)